src/main/java/org/znerd/confluence/client/http/ConfluenceLabel

```diff
@@ -0,0 +1,50 @@
+package org.znerd.confluence.client.http;
+
+public class ConfluenceLabel {
```

Comment:
Some suggestions:
- Mark the class as `final`.
- Make the fields read-only: remove the no-arg constructor, remove the setters, mark the fields as `final`.
- Add `equals()` and `hashCode()` methods (can be generated automatically by e.g. IntelliJ IDEA).
- Add a unit test that verifies the latter 2 methods.

Sample unit test methods:
```
@Test
  void equalsAndHashCode() {
    EqualsVerifier.forClass(ConfluenceLabel.class).verify();
  }

  @Test
  void toStringMethod() {
    ToStringVerifier.forClass(ConfluenceLabel.class).verify();
  }
```

Reply:
Hmm, I just realized the code base is still using an older version of JUnit, while my sample is based on JUnit 5. So feel free to ignore that.